README.md

```diff
@@ -12,5 +12,5 @@ data_vault.py               Store numeric data from experiments, supported by th
 data_vault_multihead.py     Data vault implementation that can connect to multiple LabRAD managers
 
 Some of the tools here require PyQt.
-To install PyQt into a virtualenv in Linux, see [these instructions](https://docs.google.com/document/d/1f_rZinhOOSVE_wjScR3jwJagt6MIqkIfthttmF7RWtk).
-If you don't have access, please simply request.
+To install PyQt into a virtualenv in Linux, see [these instructions](https://docs.google.com/document/d/1PR0Xm6NqutMZs00Dt8U4CWlamVYgWxH52_8r-8apnVE/).
```

Comment:
We have our automated installer script in `pyle`, and the PyQt4 installer instructions in the `@google` drive refers to that script. On the other hand, the version of the instructions in my personal drive doesn't refer to the script. We should clean this all up, but we can't give everyone access to `pyle`. Thoughts?